gazelle/docs/installation_and_usage.md

```diff
@@ -0,0 +1,156 @@
+# Installation and Usage
+
+## Example
+
+Examples of using Gazelle with Python can be found in the `rules_python`
+repo:
+
+* bzlmod: {gh-path}`examples/bzlmod_build_file_generation`
+* WORKSPACE: {gh-path}`examples/build_file_generation`
+
+:::{note}
+The following documentation covers using bzlmod.
+:::
+
+
+## Adding Gazelle to your project
+
+First, you'll need to add Gazelle to your `MODULE.bazel` file. Get the current
+version of Gazelle from their [releases page][gazelle-releases].
+
+[gazelle-releases]: https://github.com/bazel-contrib/bazel-gazelle/releases/
+
+See the installation `MODULE.bazel` snippet on the `rules_python`
+[releases page][rules-python-releases] in order to configure `rules_python`.
+
+[rules-python-releases]: https://github.com/bazel-contrib/rules_python/releases
+
+You will also need to add the {bzl:obj}`bazel_dep` for configuration for
+`rules_python_gazelle_plugin`.
+
+Here is a snippet of a `MODULE.bazel` file.
+
+```starlark
+# The following stanza defines the dependency rules_python.
+bazel_dep(name = "rules_python", version = "0.22.0")
+
+# The following stanza defines the dependency rules_python_gazelle_plugin.
+# For typical setups you set the version.
+bazel_dep(name = "rules_python_gazelle_plugin", version = "0.22.0")
+
+# The following stanza defines the dependency gazelle.
+bazel_dep(name = "gazelle", version = "0.31.0", repo_name = "bazel_gazelle")
```

Comment:
These are very old, is it on purpose?

EDIT: this is just a move, it's fine to leave it as it is.